packages/api/test/__fixtures__/sdk/petstore/index.ts

```diff
@@ -448,6 +448,8 @@ export interface Pet {
   tags?: Tag[];
   /**
    * pet status in the store
+   *
+   * `available` `pending` `sold`
```

Comment:
These are here now because I bumped `oas` which included https://github.com/readmeio/oas/pull/637 that adds enums to response property descriptions.